rdr_service/tools/generate_swoop_data.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+
+# Generates swoop & n3c query data from rdr and puts in a bucket for data analysis team
+
+USAGE="tools/generate_swoop_data.sh [--account <ACCOUNT> --project <PROJECT> [--creds_account <ACCOUNT>]]"
+while true; do
+  case "$1" in
+    --account) ACCOUNT=$2; shift 2;;
+    --creds_account) CREDS_ACCOUNT=$2; shift 2;;
+    --project) PROJECT=$2; shift 2;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
+
+if [ "${PROJECT}" ]
+then
+  if [ -z "${ACCOUNT}" ]
+  then
+    echo "Usage: $USAGE"
+    exit 1
+  fi
+  if [ -z "${CREDS_ACCOUNT}" ]
+  then
+    CREDS_ACCOUNT="${ACCOUNT}"
+  fi
+  source tools/auth_setup.sh
+  run_cloud_sql_proxy
+  set_db_connection_string
+else
+  if [ -z "${DB_CONNECTION_STRING}" ]
+  then
+    source tools/setup_local_vars.sh
+    set_local_db_connection_string
+  fi
+fi
+
+source tools/set_path.sh
+python tools/generate_swoop_data.py --project $PROJECT
```

Comment:
We shouldn't have a need for this, when adding a tool we want to follow the construct of the tools in the `tool_libs` directory which gives you all you need for setting up tools and inheriting from the `ToolBase` class.
ex: `rdr_service/tools/tool_libs/hpro_consent.py`

Reply:
Thank you, I will check and resolve this